packages/connectivity/connectivity_web/ios/Classes/SwiftConnectivityWebPlugin.swift

```diff
@@ -0,0 +1,14 @@
+import Flutter
```

Comment:
You don't actually need the swift class. Just the podspec should be enough/

Reply:
Cool, I'll clean this up! This came from the plugin template as well